app/s3_client/s3_csv_client.py

```diff
@@ -10,13 +10,13 @@
 )
 from notifications_utils.s3 import s3upload as utils_s3upload
 
-FILE_LOCATION_STRUCTURE = "service-{}-notify/{}.csv"
+NEW_FILE_LOCATION_STRUCTURE = "{}-service-notify/{}.csv"
```

Comment:
Do we not need to keep the old way of referencing objects like we do in the API?

Reply:
No, because on the admin side this file structure is only used for 'new' sends.  All of the looking up of old sends is done on the API side.

That being said, we can't test until the quota issue is solved.  I wouldn't want to push this up to production without being sure it worked. :)

Reply:
Understood, thanks! 🙂